key.js

```diff
@@ -241,6 +488,8 @@ const homeAnimation = () => {
 }
 
 const startAnimation = () => {
+    defaultLife.textContent = `  x ${lives}`;
```

Comment:
No need of spaces here.  Use CSS to style and add spaces in the HTML source file, or dynamically through controller logic by adding a class that contains the required CSS.

```suggestion
    defaultLife.textContent = `x ${lives}`;
```